pss/pss_test.go

```diff
@@ -166,39 +130,38 @@ func TestCache(t *testing.T) {
 	data := []byte("foo")
 	datatwo := []byte("bar")
 	datathree := []byte("baz")
-	mparams := &messageParams{
-		Src:     privkey,
-		Dst:     &privkey.PublicKey,
-		Topic:   PingTopic,
-		Payload: data,
+	wparams := &crypto.WrapParams{
+		Src: privkey,
+		Dst: &privkey.PublicKey,
 	}
-	env, err := newSentEnvelope(mparams)
-	msg := &PssMsg{
+	env, err := ps.Crypto.WrapMessage(data, wparams)
+	msg := &message.Message{
 		Payload: env,
 		To:      to,
+		Topic:   PingTopic,
 	}
-	mparams.Payload = datatwo
-	envtwo, err := newSentEnvelope(mparams)
-	msgtwo := &PssMsg{
+	envtwo, err := ps.Crypto.WrapMessage(datatwo, wparams)
+	msgtwo := &message.Message{
 		Payload: envtwo,
 		To:      to,
+		Topic:   PingTopic,
 	}
-	mparams.Payload = datathree
-	envthree, err := newSentEnvelope(mparams)
-	msgthree := &PssMsg{
+	envthree, err := ps.Crypto.WrapMessage(datathree, wparams)
+	msgthree := &message.Message{
 		Payload: envthree,
 		To:      to,
+		Topic:   PingTopic,
 	}
 
-	digestone := ps.msgDigest(msg)
+	digestone := msg.Digest()
 	if err != nil {
 		t.Fatalf("could not store cache msgone: %v", err)
```

Comment:
These error messages suggest something is _stored_, but that's not the case any longer, I believe.

Reply:
Again, not part of the changes in this PR.